compiler-plugin-tests/src/test/resources/ballerina_sources/validator_tests/28_invalid_input_types/Ballerina.toml

```diff
@@ -2,3 +2,8 @@
 org = "graphql_test"
 name = "test_package"
 version = "0.1.0"
+
+[[dependency]]
+org = "ballerina"
+name = "http"
+version = "2.5.2"
```

Comment:
This is not needed.

Reply:
I have added this to avoid modifying the fully qualified name every-time the version changes. Seems this is not working as expected. Will update the code to use the types from a sub module

Reply:
Fixed with [1bfba27](https://github.com/ballerina-platform/module-ballerina-graphql/pull/1149/commits/1bfba27b7d1fed3210f77fb8bafbc52481bf5633)

Reply:
Yeah, that will not work, and this possibly makes future releases hard. We might have to change this in each release. Let's try to find an alternative.